docs/tutorials/vae.rst

```diff
@@ -29,32 +29,31 @@ Build the Model
 ---------------
 
 In ZhuSuan, a model is constructed under a
-:class:`~zhusuan.model.base.BayesianNet` context, which enables transparent
+:class:`~zhusuan.framework.bn.BayesianNet` context, which enables transparent
```

Comment:
now it is not a context? maybe "constructed using :class:`~zhusuan.framework.bn.BayesianNet`", and have you make sure all these internal reference links worked?